docs/4.3/kubernetes_ssh.md

```diff
@@ -137,10 +176,22 @@ configured. In this guide we'll look at two common configurations:
 
 ## Kubernetes Groups and Users
 
-Teleport provides support for Kubernetes Group `kubernetes_groups: ["system:masters"]`
-and Kubernetes Users, using `kubernetes_users: ['barent', 'jane']`. If a Kubernetes
+Teleport provides support for
+
+* Kubernetes Groups, using `kubernetes_groups: ["system:masters"]`.
+* Kubernetes Users, using `kubernetes_users: ['barent', 'jane']`. If a Kubernetes
 user isn't set the user will impersonate themselves.
 
+When adding new local users you have to specify which Kubernetes groups they
+belong to:
+
+``` bash
+# Adding a Teleport local user to map to a Kubernetes group.
+$ tctl users add joe --k8s-groups="system:masters"
+# Adding a Teleport local user to map to a Kubernetes user.
+$ tctl users add jenkins --k8s-users="jenkins"
+```
```

Comment:
Add the same note under this about this being OSS only, and say that Enterprise users can manage `k8s-users` and `k8s-groups` via RBAC.